src/index.html

```diff
@@ -11,12 +11,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo_img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="list">
+          <li class="list-item">
+            <a
+              href="#"
+              class="link is-active"
```

Comment:
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```